import/plugin.h

```diff
@@ -44,6 +47,24 @@ class RawWebView : public QuickMozView
     ~RawWebView();
 };
 
+// using custom translator so it gets properly removed from qApp when engine is deleted
+class AppTranslator: public QTranslator
```

Comment:
While this is small, we could move this to Silica C++ library. Currently every plugin has a copy.